actions/actionsRouter.js

```diff
@@ -6,4 +6,18 @@ const knexConfig = require("../knexfile");
 const router = express.Router();
 const db = knex(knexConfig.development);
 
+router.post("/", (req, res) => {
+  const { description, notes, completed, project_id } = req.body;
+  if (description && notes && !completed && project_id) {
+    db("actions")
+      .insert(req.body)
```

Comment:
Instead of inserting the entire request body, specify which properties you want. This will stop malicious attacks to your database.